### ISSUE TITLE
can not reproduce the IOU result

### ISSUE THREAD

**Issue by CODE-SUBMIT** [CubiCasa/CubiCasa5k#16]
Dear 
 I run the code, however, I can not reproduce the result in the paper,  Is there anything else I could do to reproduce the results

**Comment by morrisalp** [comment]
Something seems strange with the learning rate scheduling, as the LR drops to 1e-5 pretty early while I get better results from continuing to train at 1e-4.

**Comment by VietPT3502** [comment]
> Something seems strange with the learning rate scheduling, as the LR drops to 1e-5 pretty early while I get better results from continuing to train at 1e-4.

hello, i have tried the default settings and try to dont drop lr too early but my result are still bad. What are ur result. and can you show me your evaluation and training log
